analyzer/src/main/java/org/jboss/gm/analyzer/alignment/AlignmentService.java

```diff
@@ -101,7 +122,7 @@ default int order() {
 
         ResponseCustomizer NOOP = new ResponseCustomizer() {
             @Override
-            public Response customize(Response response) {
+            public Response customize(Response response, Request request) {
```

Comment:
I'm really not convinced this is needed. I think we need to take another look at how to order and lay this out

Reply:
What do you have in mind? Why does this not look an appropriate solution my I ask?

Reply:
Oh while it does look like it would work I just wonder whether there is a simpler option - for instance maintaining an extra (non-serialised) map in the model which temporarily is used to store the originalKey->resolvedKey (which is passed to DA) .... essentially I'm just thinking about whether we keep things simpler :-)

Reply:
You are probably right! I'll take a look at it with a fresh mind tomorrow 😉

Reply:
Cool, will have a think as well, I think we've made good progress establishing a baseline of what is needed.

Reply:
Agreed 😎